.github/workflows/fossa.yml

```diff
@@ -0,0 +1,17 @@
+name: Fossa CLI
+on:
+  push:
+    branches:
+      - 'develop'
+      - 'release/**'
```

Comment:
No release/** branches on this repo. Doesn't hurt though.

Reply:
removed